front/style/layout/_layout.scss

```diff
@@ -335,6 +335,10 @@
             margin-top: 40px;
         }
 
+        &--top-30 {
```

Comment:
`top-30` and `top-large` have the same values. We should be using one, and if it's the `top-30`, then replace all the `top-large` in the code, and delete the class ! :)